app/decorators/controllers/hyrax/admin/stats_controller_decorator.rb

```diff
@@ -0,0 +1,11 @@
+module Hyrax
+  Admin::StatsController.class_eval do
+    def show
+      super
+      @works_to_display ||= ::FileDownloadStat.all.select do |e|
+        e[:user_id] == current_user.id
+      end
+      @total_downloads ||= @works_to_display.map(&:downloads).reduce(:+)
```

Comment:
why are you using `||=` in this method? Is there somewhere else that could have already set `@total_downloads`?  If so, can you add a comment about where that might occur?  Otherwise just use simple assignment (`=`)